src/stim/diagram/detector_slice/detector_slice_set.cc

```diff
@@ -615,31 +617,59 @@ void DetectorSliceSet::write_svg_diagram_to(std::ostream &out) const {
 
     for (size_t k = 0; k < num_ticks; k++) {
         for (auto q : used_qubits()) {
-            auto c = coords(min_tick + k, q);
+            auto t = min_tick + k;
+            
+            std::stringstream id_ss;
+            id_ss << "qubitdot";
+            id_ss << ":" << q;
+            bool first = true;
+            for (auto c : coordinates.at(q)){
+                if(first){
+                    id_ss << ":";
+                } else {
+                    id_ss << "_";
+                }
+                id_ss << c; // the raw qubit coordinates, not projected to 2D
+                first = false;
+            }
+            id_ss << ":" << t; // the absolute tick
+            
+            auto sc = coords(t, q); // the svg coordinates, offset to the correct slice plot
+
             out << "<circle";
-            write_key_val(out, "cx", c.xyz[0]);
-            write_key_val(out, "cy", c.xyz[1]);
+            write_key_val(out, "id", id_ss.str());
+            write_key_val(out, "cx", sc.xyz[0]);
+            write_key_val(out, "cy", sc.xyz[1]);
             write_key_val(out, "r", 2);
             write_key_val(out, "stroke", "none");
             write_key_val(out, "fill", "black");
             out << "/>\n";
         }
     }
 
-    // Boundaries around different slices.
+    // Border around different slices.
     if (num_ticks > 1) {
+        size_t k = 0;
         for (uint64_t col = 0; col < num_cols; col++) {
             for (uint64_t row = 0; row < num_rows && row * num_cols + col < num_ticks; row++) {
                 auto sw = coordsys.size.xyz[0];
                 auto sh = coordsys.size.xyz[1];
+                   
+                std::stringstream id_ss;
+                id_ss << "border:" << k;
```

Comment:
Maybe `tick_border`?

What's the reasoning for including both `k` and `k+min_tick`?

Reply:
I started including k to make it easy to iterate through them, and later made it so all of them had the absolute tick at the end for similar reasons; happy to remove k if you like

Reply:
Na, you're the key user so you have the best sense of what's useful.